include/manif/impl/se3/SE3Tangent_base.h

```diff
@@ -125,7 +125,7 @@ struct SE3TangentBase : TangentBase<_Derived>
     return Eigen::Map<SO3Tangent<Scalar>>(coeffs.data()+3);
   }
 
-private:
+// private:
```

Comment:
@artivis  Is it ok that I commented out the `private` specifier here? Do you recommend that I remove it?

Reply:
You can leave it as it is, I am planning on [moving this function outside the class in the future](https://github.com/artivis/manif/blob/d33e100dd0acd9d6f48ecff69169114481aa8656/include/manif/impl/se3/SE3_utils.h#L8).